features/admin.groups.v1/components/wizard/create-group-wizard.tsx

```diff
@@ -167,34 +192,20 @@ export const CreateGroupWizard: FunctionComponent<CreateGroupProps> =
 
             setSubmitStep(WizardStepsFormTypes.BASIC_DETAILS);
         }
+    }, []);
 
-        if (roleList?.length < 1) {
-            if (isFirstLevelOrganization()) {
```

Comment:
What is the reason for removing this logic?

Reply:
This is no longer needed. Moving forward `scim2` endpoint should be used. `organizationRoles` endpoint is not used. Mentioned in the issue description.